api/scpca_portal/test/models/test_job.py

```diff
@@ -18,6 +18,40 @@ def setUp(self):
             f"{self.mock_project_id}-{self.mock_download_config_name}"
         )
         self.mock_dataset = DatasetFactory()
+        self.aws_batch_job_status = [  # AWS Batch job statuses
+            "SUBMITTED",
+            "PENDING",
+            "RUNNABLE",
+            "STARTING",
+            "RUNNING",
+            "SUCCEEDED",
+            "FAILED",
+        ]
+
+    def bulk_create_mock_jobs(self, list_of_jobs):
+        """Helper to create job instances using JobFactory."""
+        for job in list_of_jobs:
+            JobFactory(
+                batch_job_name=self.mock_project_batch_job_name,
+                batch_job_id=job["batch_job_id"],
+                state=job["state"],
+            )
+
+    def bulk_create_mock_aws_batch_jobs(self, list_of_jobs, terminated_job=None):
```

Comment:
```suggestion
    def create_mock_aws_batch_jobs(self, list_of_jobs, terminated_job=None):
```

Dont really need to worry about "bulk" in this context as the plurality of "jobs" covers that.